item-management/lastpass-vault-item-import.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+#This script will create vaults and login items from a LastPass export. 
+
+#Shared/Nested folders in 1Password will have separate vaults created. Items not belonging to any shared folder will be created in the user's private vault.
+
+
+
+import csv, os
+vault_list = []
+with open('export.csv', newline='') as csvfile:
+    linereader = csv.reader(csvfile, delimiter=',', quotechar='"')
+    next(linereader)
+
+    
+    for row in linereader:
+        url = row[0]
+        username = row[1]
+        password = row[2]
+        notes= row[4]
+        title = row[5]        
+        vault = row[6]
+        
+        #omitting Secure Notes
+        if url == "http://sn":
+            continue
+        
+        if not vault or vault == "":
+            os.system('''op item create --vault="Private" \\
+                --tags="%s" \\
+                --category=login \\
+                --title="%s" \\
+                --url="%s" \\
+                username="%s" \\
+                password="%s" \\
+                notes="%s"
+                ''' % (vault, title, url, username,password, notes))
+            continue
+        if vault not in vault_list:
+            vault_list.append(vault) 
+            #create vault
+            os.system('op vault create "%s"'% vault)
+            #create item
+            os.system('''op item create --vault="%s" \\
+                --tags="%s" \\
```

Comment:
Currently tags get the vault (or "grouping" from the export). I wonder if, until we get OTP secrets loading in properly, we might want to make a conditional that adds a tag like `migrate 2fa` or some other value that a person can filter on so they can see all items with TOTP that need manual migration from LP? 

Something like: if the item has a value in TOTP field, tag with `migrate 2fa`, otherwise tag with `<vault name>`. Even better if we can concatenate the 2fa and vaults tags.

Reply:
I'll take a stab at that but I think I'd like to merge this is without that feature just to get it out there, since that's more of a "nice to have" than a "must have".